tests/asm/zvma/zvma.S

```diff
@@ -0,0 +1,91 @@
+.text
+.balign 16
+.globl test
+
+test:
+    addi    sp, sp, -32
+    sw      s0, 0(sp)
+    sw      s1, 4(sp)
+    sw      s2, 8(sp)
+    sw      s3, 12(sp)
+    sw      s4, 16(sp)
+    sw      s5, 20(sp)
+    sw      s6, 24(sp)
+    sw      s7, 28(sp)
+    j   test_1
+
+test_1:
+    # vtype=0b_11_0_00_000_000, twiden=4, alt=0, sew=8
+    vsetvli s1, x0, 0x600
+
+    # s1 shall be vl=TEW
+
+    .insn i OP_V, 7, s2, s1, 0x841 - 0x1000 # settm s2, s1
```

Comment:
Wait for SiFive's patches or upstreaming of https://github.com/llvm/llvm-project/pull/132965